examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -2522,6 +2542,66 @@ rtError pxScene2d::getAvailableApplications(rtString& availableApplications)
   return RT_OK;
 }
 
+#ifdef PXSCENE_SUPPORT_STORAGE
```

Comment:
Please ensure that all methods are declared and defined without surrounding #ifdefs.  Please add the #ifdef checks inside the methods themselves.  If the define is not declared then please log a message but do nothing.  However, a method should be declared if the define is set or not.  This will prevent us from having to add new defines when building other software components that use Spark headers.  Please update for all other new methods as well.